### PR TITLE
Rewrite control-label

### LINKED ISSUE #189: Rewrite control-label component
### Criteria:

- TypeScript
- Functional component
- Radix inner workings. Here's the API Reference for label: https://www.radix-ui.com/docs/primitives/components/label#api-reference

### PR DESCRIPTION
**Pull request by tristen**
Simple changes to the control-label component to use TypeScript, React testing library, and accessibility enhancements provided by [`@radix-ui/react-label`](https://www.radix-ui.com/docs/primitives/components/label#api-reference).

---

Closes #189

### CODE REVIEW

File: src/components/tabs/index.ts [hunk 1]
```diff
@@ -1,3 +0,0 @@
-import main from './tabs';
-
-export default main;
```
Comment (tristen) [inline src/components/tabs/index.ts:3]: Just noticed this unnecessary file as `src/components/tabs/index.js` exists

File: src/components/icon/icon.test.tsx [hunk 2]
```diff
@@ -8,8 +8,8 @@ jest.mock('../utils/get-window');
 // eslint-disable-next-line @typescript-eslint/no-explicit-any
 const getWindowMock = getWindow as any;
 
-describe('renders', () => {
-  test('basic', () => {
+describe('basic', () => {
+  test('renders', () => {
```
Comment (tristen) [inline src/components/icon/icon.test.tsx:12]: Small naming change to an unrelated component. I noticed the two were in the wrong order.

File: src/components/control-label/examples/control-label-example-basic.tsx [hunk 3]
```diff
@@ -0,0 +1,14 @@
+/*
+Basic.
+*/
+import React from 'react';
+import ControlLabel from '../control-label';
+
+export default function Example() {
+  return (
+    <>
+      <ControlLabel id="name" text="Enter your name" />
```
Comment (miafan23) [inline src/components/control-label/examples/control-label-example-basic.tsx:10]: Maybe replace the value of `id` with another one? It is the same as the id of `controlFile`, so when I click this label on the example page, the dialog for selecting files popped up.

**Review by miafan23 — APPROVED** [review]
LGTM! Only one minor fixing might be required.

File: src/components/control-label/examples/control-label-example-basic.tsx (hunk 3 above)
Comment (tristen) [reply src/components/control-label/examples/control-label-example-basic.tsx:10]: ohhh nice catch! I might change this in https://github.com/mapbox/mr-ui/pull/186 then as it seems strange for id to be `name` there.

**Pull request merged**